src/status_im/data_store/pin_messages.cljs

```diff
@@ -29,7 +29,8 @@
 
 (rf/defn send-pin-message
   [cofx pin-message]
-  {:json-rpc/call [{:method     "wakuext_sendPinMessage"
-                    :params     [(messages/->rpc pin-message)]
-                    :on-success #(log/debug "successfully pinned message" pin-message)
-                    :on-error   #(log/error "failed to pin message" % pin-message)}]})
+  {:json-rpc/call [{:method      "wakuext_sendPinMessage"
+                    :params      [(messages/->rpc pin-message)]
+                    :js-response true
+                    :on-success  #(rf/dispatch [:sanitize-messages-and-process-response %])
+                    :on-error    #(log/error "failed to pin message" % pin-message)}]})
```

Comment:
Is it okay to log the message content?

Reply:
pin-message should only have info about message-id etc, so I think it's ok in this case

Reply:
Maybe a bit better is to use `select-keys` or something, just in case the `pin-message` map changes in the future and starts to leak important data, nobody will remember to fix the log here.

Reply:
yes, good point!